openapi-cli/src/test/resources/generators/service/ballerina/requestBody/scenario_02_rb.bal

```diff
@@ -7,6 +7,6 @@ service /v1 on ep0 {
     #
     # + payload - Optional description in *Markdown*
     # + return - OK
-    resource function post pets(@http:Payload Pet|xml|map<string>|string payload) returns http:Ok {
+    resource function post pets(@http:Payload string|xml|map<string>|Pet payload) returns http:Ok {
```

Comment:
Can this change of the order of the params be a breaking change ?

Reply:
I think this won't be a breaking change since this change is about union types of order, This doesn't change parameter order.
When one request Body has multiple media types, here the List structure has used to store the list of types for creating the union type but there can be scenarios multiple mediaType can have the same Record[1] 
[1]
```openapi
"requestBody": {
                    "content": {
                        "application/json-patch+json": {
                            "schema": {
                                "$ref": "#/components/schemas/UsuarioResetSenha"
                            }
                        },
                        "application/json": {
                            "schema": {
                                "$ref": "#/components/schemas/UsuarioResetSenha"
                            }
                        },
                        "text/json": {
                            "schema": {
                                "$ref": "#/components/schemas/UsuarioResetSenha"
                            }
                        },
                        "application/*+json": {
                            "schema": {
                                "$ref": "#/components/schemas/UsuarioResetSenha"
                            }
                        }
                    }
                },
```
with the previous implementation, this has duplicate records in the union type, Therefore has to replace the List structure with the HashSet structure to avoid duplicates. That is the reason for changing the order. Hope this won't be a breaking change since we only change the types from union type, without changing parameter order.

Reply:
Ack